src/index.js

```diff
@@ -1,6 +1,6 @@
 import importRule from './rules/import';
 
-export default {
+module.exports = {
```

Comment:
Sorry, I forgot the package user does not have to install `babel-plugin-add-module-exports`. I think your suggestion is better, we don't have to mix ES6 module syntax with CommonJS syntax.